packages/ui/components/input-amount/test/lion-input-amount.test.js

```diff
@@ -128,6 +128,21 @@ describe('<lion-input-amount>', () => {
     expect(_inputNode.value).to.equal('100.12');
   });
 
+  it('parses an updated value based on locale', async () => {
+    const el = /** @type {LionInputAmount} */ (
+      await fixture(
+        html`<lion-input-amount .modelValue=${123456.78} currency="EUR"></lion-input-amount>`,
+      )
+    );
+    const { _inputNode } = getInputMembers(/** @type {* & LionInput} */ (el));
+    expect(_inputNode.value).to.equal('123,456.78');
+    _inputNode.value = '123,45';
```

Comment:
Do we expect this when we programatically add a value? Or only when the user types? (probably the latter)

Reply:
I probably works correctly as is atm (as the formatter triggered by `.modelValue`doesn't use the formatOptions atm), but the "pasted" approach (cleaning up the "preformatted" state) might be safer in general